hack/generate-manifest.sh

```diff
@@ -141,6 +141,9 @@ mkdir configMap && cd configMap
 # user is not expected to make changes directly to antrea-agent.conf but instead to the generated
 # YAML manifest, so our regexs need not be too robust.
 cp $KUSTOMIZATION_DIR/base/conf/antrea-agent.conf antrea-agent.conf
+# user is not expected to make changes directly to antrea-controller.conf but instead to the generated
+# YAML manifest, so our regexs need not be too robust.
+cp $KUSTOMIZATION_DIR/base/conf/antrea-controller.conf antrea-controller.conf
```

Comment:
nit: let's not repeat the same comment here, but instead update the comment above to reference "antrea-agent.conf & antrea-controller.conf" instead of just "antrea-agent.conf"